coordinator/internal/authority/authority_test.go

```diff
@@ -183,13 +183,13 @@ func newManifest(t *testing.T) (*manifest.Manifest, []byte, [][]byte) {
 	policy := []byte("=== SOME REGO HERE ===")
 	policyHash := sha256.Sum256(policy)
 	policyHashHex := manifest.NewHexString(policyHash[:])
-	mnfst := &manifest.Manifest{
-		Policies:                map[manifest.HexString][]string{policyHashHex: {"test"}},
-		WorkloadOwnerKeyDigests: []manifest.HexString{keyDigest},
-	}
+
+	mnfst := manifest.DefaultAKS()
+	mnfst.Policies = map[manifest.HexString][]string{policyHashHex: {"test"}}
+	mnfst.WorkloadOwnerKeyDigests = []manifest.HexString{keyDigest}
```

Comment:
If we need to change this (I don't think we need to), could we use `manifest.Default` instead?

Reply:
The default manifest does not include the reference values, so when testing `authority.SNPValidateOpts`, it would fail.

Reply:
But... it's working now without reference values, isn't it? (Admitted, there's also no test.)

Reply:
Before, the reference values would default to 0, but now they will default to `null`, in which case `manifest.SNPValidateOpts` returns an error when using `manifest.Default()`.

Reply:
Understood, thanks.